bindings/cs/rl.net/LiveModel.cs

```diff
@@ -775,6 +844,40 @@ public void QueueOutcomeEvent(string eventId, string outcomeJson)
             }
         }
 
+        public bool TryQueueOutcomeEvent(string eventId, int slotIndex, float outcome, ApiStatus apiStatus = null)
+        {
+            int result = LiveModelReportOutcomeSlotF(this.DangerousGetHandle(), eventId, slotIndex, outcome, apiStatus.ToNativeHandleOrNullptrDangerous());
+
+            GC.KeepAlive(this);
+            return result == NativeMethods.SuccessStatus;
+        }
+
+        public void QueueOutcomeEvent(string eventId, int slotIndex, float outcome)
+        {
+            using (ApiStatus apiStatus = new ApiStatus())
+                if (!this.TryQueueOutcomeEvent(eventId, slotIndex, outcome, apiStatus))
+                {
+                    throw new RLException(apiStatus);
+                }
+        }
+
+        public bool TryQueueOutcomeEvent(string eventId, int slotIndex, string outcomeJson, ApiStatus apiStatus = null)
+        {
+            int result = LiveModelReportOutcomeSlotJson(this.DangerousGetHandle(), eventId, slotIndex, outcomeJson, apiStatus.ToNativeHandleOrNullptrDangerous());
+
+            GC.KeepAlive(this);
+            return result == NativeMethods.SuccessStatus;
+        }
+
+        public void QueueOutcomeEvent(string eventId, int slotIndex, string outcomeJson)
```

Comment:
I am wondering if `Queue` is the right naming here, the fact that reporting an event sends it to a queue seems like an internal detail

Reply:
We picked this name because it better conveys the fact that it's not guaranteed that it will end up on the other end..